packages/flutter/test/widgets/raw_radio_test.dart

```diff
@@ -0,0 +1,52 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+// This file is run as part of a reduced test set in CI on Mac and Windows
+// machines.
+@Tags(<String>['reduced-test-set'])
+library;
+
+import 'package:flutter/widgets.dart';
+import 'package:flutter_test/flutter_test.dart';
+
+void main() {
+  testWidgets('RawRadio control test', (WidgetTester tester) async {
+    final FocusNode node = FocusNode();
+    addTearDown(node.dispose);
+    int? actualValue;
+    ToggleableStateMixin? actualState;
+    await tester.pumpWidget(
+      RawRadio<int>(
+        value: 0,
+        groupValue: 1,
+        onChanged: (int? value) {
+          actualValue = value;
+        },
+        mouseCursor: WidgetStateProperty.all<MouseCursor>(SystemMouseCursors.click),
+        toggleable: true,
+        focusNode: node,
+        autofocus: false,
+        builder: (ToggleableStateMixin state) {
+          actualState = state;
+          return CustomPaint(size: const Size(40, 40), painter: TestPainter());
```

Comment:
Nit: I think it would be fine to return SizedBox.shrink or Placeholder here if you want to save a few lines.